src/main/features/claim/views/amount.njk

```diff
@@ -20,13 +20,13 @@
         <div class="claim-amounts">
           <div id="claim-amount">
             <div>
-              <div id="claim-amount-heading-1" class="claim-amount-column-1 mobile-hide">
+              <div id="claim-amount-heading-1" class="claim-amount-column-1 phone-hide">
                 <span id="claim-reason-label" class="form-label-bold">{{ t('What you’re claiming for') }}</span>
                 <span id="claim-reason-hint" class="form-hint">
                   {{ t('Briefly explain each item - for example, "broken tiles", "roof damage"') }}
                 </span>
               </div>
-              <div id="claim-amount-heading-2" class="claim-amount-column-2 mobile-hide">
+              <div id="claim-amount-heading-2" class="claim-amount-column-2 phone-hide">
```

Comment:
this should remain as mobile-hide as it relates to hiding this when a user in on a mobile